data/json/itemgroups/guns.json

```diff
@@ -204,6 +205,8 @@
       { "item": "l_mbr_223", "prob": 10, "charges-min": 0, "charges-max": 32 },
       { "item": "l_long_45", "prob": 30, "charges-min": 0, "charges-max": 12 },
       { "item": "m14ebr", "prob": 15, "charges-min": 0, "charges-max": 20 },
+      { "item": "ar50", "prob": 15, "charges-min": 0, "charges-max": 1 },
+      { "item": "as50", "prob": 20, "charges-min": 0, "charges-max": 10 },
```

Comment:
Can you provide sources for the AS 50 being imported for the commercial market and available for sale through FFL's?

Reply:
i don't have it, but lots of people i ask around internet says that there is AS 50 in US.

Reply:
@RickyVanz if there are, you should be able to find one for sale somewhere. Or heck, send AI an email asking if any are available/have been imported.

If it's something that has to be special ordered, it's not going to even really be common enough to put in unless you crank everything else's weights way up.

Reply:
"Lots of people around the internet" isn't a source. If it's not used by us armed forces or police and it's not readily available to private citizens, it doesn't belong in the game.

Reply:
allright, should i delete the AS 50?

Reply:
No, leaving it in only in  the swat item group is sufficient.

Reply:
Nowhere, it spawns appropriately already. Just delete the AS50 additions you've made and leave the one in SWAT alone

Reply:
allright, that means i replace my previously merged PR with this new one..

at least now there is variety for .50 cal, that's what users on discourse forum want :smile: